examples/java/build.gradle

```diff
@@ -73,7 +73,17 @@ dependencies {
   compile library.java.joda_time
   compile library.java.proto_google_cloud_datastore_v1
   compile library.java.slf4j_api
-  compile library.java.slf4j_jdk14
+  compile "com.google.api.grpc:proto-google-cloud-language-v1:1.81.4"
+  compile ("io.confluent:kafka-avro-serializer:5.3.2") {
+    // It depends on "spotbugs-annotations:3.1.9" which clashes with current
+    // "spotbugs-annotations:3.1.12" used in Beam. Not required.
+    exclude group: "org.apache.zookeeper", module: "zookeeper"
```

Comment:
In order to remove "com.github.spotbugs:spotbugs-annotations:3.1.9", had to exclude the zookeeper from kafka-avro-serializer.